e2e/run.ts

```diff
@@ -5,7 +5,7 @@ const { AUTOBLOCKS_API_KEY, AUTOBLOCKS_INGESTION_KEY } = process.env;
 
 // We've created a view in our demo org to be used in CI tests.
 // It has one filter, message == 'sdk.e2e', and its timespan is "last 1 hour"
-const E2E_TESTS_VIEW_ID = 'clldzryfx0001i908okbbe5pf';
+const E2E_TESTS_VIEW_ID = 'cllh1o6bz0001jr08kocrn1fw';
```

Comment:
I deleted this view so the sidebar would be clear of views for the demo video, but it occurred to me I could just hide it from the sidebar in the UI